core/ledger/pvtdatastorage/store.go

```diff
@@ -406,6 +429,40 @@ func (s *Store) ResetLastUpdatedOldBlocksList() error {
 	return nil
 }
 
+func (s *Store) addPurgeMarkers(p *PurgeMarker) error {
+	b := s.db.NewUpdateBatch()
+	b.Put(
+		encodePurgeMarkerCollKey(
+			&purgeMarkerCollKey{
+				ns:   p.Ns,
+				coll: p.Coll,
+			},
+		),
+		encodePurgeMarkerVal(
+			&purgeMarkerVal{
+				blkNum: p.BlkNum,
+				txNum:  p.TxNum,
+			},
+		),
+	)
```

Comment:
This key wasn't mentioned in the RFC or your google design doc. I understand the need for it after reviewing the code, but could you update the google design doc and RFC to match?

Reply:
Yes, this is a new addition for an optimization. As this is an optimization decided at the coding time and does not make any significant change at the design, I am not sure whether change in RFC is needed. Yes, I'll update the google doc at the end to keep the list of latest data items updated at one place.